draft-ietf-httpbis-compression-dictionary.md

```diff
@@ -264,6 +264,37 @@ longest "match-query" takes precedence.
 1. Given equivalent destination, path and search precedence, the most recently
 fetched dictionary takes precedence.
 
+## Content-Dictionary
+
+When a HTTP server responds with a resource that is encoded with a dictionary
+the server SHOULD send the hash of the dictionary that was used in the
+"Content-Dictionary" response header.
```

Comment:
Usually when we use "SHOULD", there is a discussion about what sort of reasons might lead to the recommendation being ignored.  What might cause a server to choose not to include this field?

Reply:
I changed it to MUST since there isn't really a good case for when you wouldn't send the dictionary used in the response.